app/views/mail/timeLimit.scala.html

```diff
@@ -1,10 +1,21 @@
 @(userName: String, projectName: String, taskId: String, annotationId: String, uri: String)
-Time limit reached for
 
-  User: @userName
-  Project: @projectName
-  Task: @taskId
+@emailBaseTemplate() {
+  <p>Hi,</p>
+  
+  <p>
+    We noticed a user reached the time limit for a project:
+  </p>
 
-  @{uri}/annotations/Task/@annotationId
+  <p>
+    <ul>
+      <li>User: @{userName}</li>
+      <li>Project: @{projectName}</li>
+      <li>Task: @{taskId}</li>
+    </ul>
+    
+  </p>
+  <p>For mor information visit <a href="@{uri}/annotations/Task/@annotationId">@{uri}/annotations/Task/@annotationId</a></p>
```

Comment:
I am no scala string interpolation expert, but I would assume the `annotationId` should also be in curly brackets (as is the uri)? (/cc @fm3)

```suggestion
  <p>For mor information visit <a href="@{uri}/annotations/Task/@{annotationId}">@{uri}/annotations/Task/@{annotationId}</a></p>
```

Reply:
I think both work. One is a scala shorthand